google/cloud/forseti/scanner/scanners/bigquery_scanner.py

```diff
@@ -14,16 +14,22 @@
 
 """Scanner for the Big Query rules engine."""
 
+import collections
+
 from google.cloud.forseti.common.gcp_type.bigquery_access_controls import (
     BigqueryAccessControls)
 
+from google.cloud.forseti.common.gcp_type import project
 from google.cloud.forseti.common.util import logger
 from google.cloud.forseti.scanner.audit import bigquery_rules_engine
 from google.cloud.forseti.scanner.scanners import base_scanner
 
 
 LOGGER = logger.get_logger(__name__)
 
+BigqueryAccessControlsData = collections.namedtuple(
+    'BigqueryAccessControlsData',
+    ['resource', 'bigquery_acl'])
```

Comment:
(here and in function parameters of find_policy_violations). To avoid ambiguity, I think we should either rename resource to something like parent_project, or change resource to storing the dataset resource.

A possible advantage of using the dataset as the resource is it allows the bigquery scanner to examine other properties of the dataset (e.g. location) in the future.

Reply:
I went with option 1 since there is no gcp_type Dataset defined yet, and I don't want to make this PR even bigger :) We can change it fairly easily down the line if it's needed!